cmd/karavictl/cmd/tenant.go

```diff
@@ -121,3 +124,11 @@ func jsonOutput(w io.Writer, v interface{}) error {
 	}
 	return nil
 }
+
+// jsonOutput() omits boolean flag on false value while encoding
+func jsonOutputModified(w io.Writer, m protoreflect.ProtoMessage) error {
```

Comment:
Can this be renamed to `jsonOutputEmitEmpty`? That is more specific to what this does.

Reply:
Renamed